index.html

```diff
@@ -114,6 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Vangelis Kontogiannis</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank">@Vangelis_Kont</a>
+          </p>
+          <p class="about">Currently, I love learning about operating systems and database systems</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="This is a link to a very cool youtube channel">
+                  https://www.youtube.com/c/Fireship
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="This link contains all the programming knowledge of the world XD">
+                  https://www.youtube.com/c/Freecodecamp
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="This last link happens to be the one that I use the most">
+                  https://en.cppreference.com/w/
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment to
`<!-- ________ Your Name Contributor card END ________ -->`